extensions/portfolio-report/manifest.json

```
@@ -0,0 +1,2601 @@
+{
+  "version": 1,
+  "locale": "en_US",
+  "platform": "4.4.3",
+  "metadata": {
+    "appmode": "rmd-static",
+    "primary_rmd": "portfolio-report-email.Rmd",
```

```suggestion
    "primary_rmd": "portfolio-report.Rmd",
```

@fizzboop - this may be the root of the problem.

---

Ah, I see. Thank you for the suggestion!

---

I'm not sure why the email Rmd was chosen. I'm mostly OOO this week, but @toph-allen can help you understand how the manifest was generated (assuming it was done with `rsconnect`). It's possible that there's an issue in rsconnect or we needed to provide an argument when generating the manifest.